In [2]:
name = "Christina"
hourly_wage = 5.35
hours = 40
weekly_pay = hourly_wage * hours
print(f"{name} worked {hours} this week and since they earn {hourly_wage} dollars per hour, they earned {weekly_pay} dollars this week.")

Christina worked 40 this week and since they earn 5.35 dollars per hour, they earned 214.0 dollars this week.


In [3]:
temps = [61, 65, 72, 58, 70, 74, 68]
tempsAboveSixtyFive = 0
for t in temps:
    if t > 65:
        tempsAboveSixtyFive = tempsAboveSixtyFive + 1
print(tempsAboveSixtyFive)

4


In [10]:
temp = [61, 65, 72, 58, 70, 74, 68]

def to_celsius(f):
    return (f - 32) * 5/9

print(round(to_celsius(temp[0]), 1))

16.1


In [11]:
movies = {"title": "Dr. Strangelove Or: How I Learned to Stop Worrying and Love the Bomb", "year": 1964, "rating": 8.3}
for key, value in movies.items():
    print(key, ":", value)

title : Dr. Strangelove Or: How I Learned to Stop Worrying and Love the Bomb
year : 1964
rating : 8.3


In [27]:
import pandas as pd
df = pd.read_csv("CS82A/chickwts (1).csv")
df.head()

,Unnamed: 0,weight,feed
0,1,179,horsebean
1,2,160,horsebean
2,3,136,horsebean
3,4,227,horsebean
4,5,217,horsebean


In [28]:
import pandas as pd
df = pd.read_csv("CS82A/chickwts (1).csv")
df.shape

(71, 3)

In [29]:
import pandas as pd
df = pd.read_csv("CS82A/chickwts (1).csv")
df.describe()

,Unnamed: 0,weight
count,71.000000,71.000000
mean,36.000000,261.309859
std,20.639767,78.073700
min,1.000000,108.000000
25%,18.500000,204.500000
50%,36.000000,258.000000
75%,53.500000,323.500000
max,71.000000,423.000000


The describe() output tells us that the average weight of the chickens was 261.31 pounds, with the lightest chicken weighing 108lbs and the heaviest weighing 423lbs. The average (mean) weight of 261.31 pounds was very close to the median of 258lbs, which typically means there was a pretty even distribution of weight between the chickens without any crazy outliers. I was surprised to see that with such a large spread between the lightest and heaviest - there was a difference of 315lbs between them. 